DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -325,7 +325,9 @@ def set_ethernet_traffic(self, device, name, pkt_data, traffic_type):
                 config_element = ti.ConfigElement.find(EndpointSetId=ep + 1)
                 # set the rate
                 config_element.FrameRate.update(
-                    Type="framesPerSecond", Rate=pkt_data.get("rate", "100")
+                    Type=pkt_data.get("frame_type", None) or "framesPerSecond",
+                    #  frame_type: "bitsPerSecond", "framesPerSecond","interPacketGap", "percentLineRate"
```

Comment:
Is there a place where we can document allowed values?

Reply:
No, not at the moment